src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java

```diff
@@ -47,7 +46,7 @@ void testGetUnNotifiedHearingsWithOutStartDateTo() {
         assertNotNull(expected.getContent());
         assertEquals(2, expected.getContent().size());
         assertEquals(2, expected.getTotalElements());
-        assertTrue(expectedHearingIds.containsAll(expected.getContent()));
+        assertEquals(expectedHearingIds, expected.getContent());
```

Comment:
Not sure about this. Isn't this the test that fails randomly because the results come back in a different order?